tests/pfcwd/files/pfcwd_helper.py

```diff
@@ -225,10 +225,20 @@ def select_test_ports(test_ports):
         selected_ports (dict): random port info or set of ports matching seed
     """
     selected_ports = dict()
+    rx_ports = set()
     seed = int(datetime.datetime.today().day)
-    for key, value in test_ports.items():
-        if (int(value['test_port_id']) % 15) == (seed % 15):
-            selected_ports.update({key:value})
+    for port, port_info in test_ports.items():
```

Comment:
Since the selected test ports basically use the same rx ports, the filtered-out ports should not be that much. In Arista7260, there should be 8 ports selected without this filter and 7 ports left with this filter.

Reply:
Yes, your approach is feasible, the sort is more like a topology sort since ports have dependencies. So the key here is whether taking1 or 2 ports out of test ports matters.

Reply:
Let's go with this approach and monitor the results